template.yaml

```diff
@@ -85,6 +85,13 @@ Conditions:
   DeployAlarms: !Or
     - Condition: IsNotDevelopment
     - !Equals [!Ref DeployAlarmsInDev, true]
+  IsCloudFrontPilotTesting: !Equals
+    - !Ref Environment
+    - dev
+  # IsCloudFrontPilotTesting: !Or
+  #   - !Equals [ !Ref Environment, build ]
+  #   - !Equals [ !Ref Environment, staging ]
+  #   - !Equals [ !Ref Environment, integration ]
```

Comment:
Will this be uncommented in the future of should it be removed?

Reply:
As the name indicates, its only for pilot and will remove the condition after deploying on all environments

Reply:
Yes but you now have 2 things named IsCloudFrontPilotTesting are both needed right now?

Reply:
Resolved